etc/ciao/ciao.json

```diff
@@ -0,0 +1,9 @@
+{
+    "server": "localhost3",
```

Comment:
Please use a more realistic (Even though fake) URL. e.g. ciao-scheduler.example.com to let people understand that this should be a real URL or an IP. No one is ever going to run a real cluster on a single machine.